vertical-tabbrowser.xml

```diff
@@ -136,39 +139,42 @@
           }
 
           let screenPos = event.screenY;
-          if (screenPos == draggedTab._dragData.animLastScreenPos)
+          if (screenPos === draggedTab._dragData.animLastScreenPos) {
             return;
+          }
 
-          let draggingForward = screenPos > draggedTab._dragData.animLastScreenPos;
           draggedTab._dragData.animLastScreenPos = screenPos;
 
-          let rtl = false; //(window.getComputedStyle(this).direction == "rtl");
+          let rtl = false; //(window.getComputedStyle(this).direction === 'rtl');
           let pinned = draggedTab.pinned;
           let numPinned = this.tabbrowser._numPinnedTabs;
           let tabs = this.tabbrowser.visibleTabs
                                     .slice(pinned ? 0 : numPinned,
                                            pinned ? numPinned : undefined);
-          if (rtl)
+          if (rtl) {
             tabs.reverse();
+          }
+
           let tabSize = draggedTab.getBoundingClientRect().height;
 
           // Move the dragged tab based on the mouse position.
 
           let leftTab = tabs[0];
           let rightTab = tabs[tabs.length - 1];
           let tabScreenPos = draggedTab.boxObject.screenY;
-          // console.log("tabScreenPos", tabScreenPos);
+          // console.log('tabScreenPos', tabScreenPos);
           let translateVal = screenPos - draggedTab._dragData.screenY;
-          // console.log("translateVal 1", translateVal);
-          if (!pinned)
+          // console.log('translateVal 1', translateVal);
+          if (!pinned){
             translateVal += this.mTabstrip.scrollPosition - draggedTab._dragData.scrollX; // XXX verticalTabs always scrollX here, it's actually the scrollPosition
+          }
           let minBound = leftTab.boxObject.screenY - tabScreenPos;
           let maxBound = (rightTab.boxObject.screenY + rightTab.boxObject.height) -
                            (tabScreenPos + tabSize);
           translateVal = Math.max(translateVal, minBound);
           translateVal = Math.min(translateVal, maxBound);
-          draggedTab.style.transform = "translateY(" + translateVal + "px)";
-          // console.log("draggedTab transform: " + draggedTab.style.transform + " should be " + "translateY(" + translateVal + "px)");
+          draggedTab.style.transform = 'translateY(' + translateVal + 'px)';
```

Comment:
Not for this pull request, but we should look into turning these into [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)…